src/components/TipInput.vue

```diff
@@ -13,14 +13,23 @@
           v-if="!tipUrlStats.tokenTotalAmount.length || +tipUrlStats.totalAmountAe !== 0"
           :amount="tipUrlStats.totalAmountAe"
         />
+      </template>
+    </Component>
+    <Dropdown
+      v-if="tipUrlStats && tipUrlStats.tokenTotalAmount.length"
+      :options="tipUrlStats.tokenTotalAmount"
+      show-right
+      class="token-tips"
+      @click.stop
```

Comment:
It is unclear in which case it is needed, can't it be done by a parent component?

Reply:
It wasn't actually needed in this particular instance.